src/containers/RouterLink/RouterLink.js

```diff
@@ -16,9 +16,7 @@ const RouterLink = React.forwardRef(function RouterLink(props, ref) {
 
   return (
     <Link as={as} href={href} replace={replace} scroll={scroll} shallow={shallow} passHref>
```

Comment:
The `ref` prop was dropped here but perhaps we can remove `React.forwardRef` entirely from this component as it's a container @alexanderflink